src/widget/tool/chattextedit.cpp

```diff
@@ -69,3 +70,54 @@ void ChatTextEdit::retranslateUi()
 {
     setPlaceholderText(tr("Type your message here..."));
 }
+
```

Comment:
Could there be added a comment saying that it's needed because Qt lacks proper translations of context menus?